articles/azure-functions/functions-reference-powershell.md

```diff
@@ -462,6 +462,21 @@ When you create a new PowerShell functions project, dependency management is ena
 
 When you update the requirements.psd1 file, updated modules are installed after a restart.
 
+If you want to use a specific version of a module, ie an older version of Az.Accounts then the one included in the included Az module, you need to add an import statement to the top of your profile.ps1 file like this:
```

Comment:
Typo: "...an older version of Az.Accounts **then** the one...", should be "**than**"

Reply:
"...the one **included** in the **included** Az module..." - repeating "**included**". Perhaps replace the first instance with "**contained**"?

Reply:
Thanks @AnatoliB, I think I'll grab the customer's commit and fix it up in the private repo.